praw/models/user.py

```diff
@@ -117,7 +117,7 @@ def me(
     ) -> Optional[Redditor]:  # pylint: disable=invalid-name
         """Return a :class:`.Redditor` instance for the authenticated user.
 
-        In :attr:`~praw.Reddit.read_only` mode, this method returns ``None``.
+        It is an error to call this method in :attr:`~praw.Reddit.read_only` mode.
```

Comment:
Can you specify exactly what it raises?

Reply:
It’s not important what it raises since it’s not an exception that should be caught.

(And if users are curious it’s not particularly difficult to try and see what happens themselves.)

Reply:
Fair point.